eth/vm/logic/storage.py

```diff
@@ -122,8 +122,10 @@ def net_sstore(gas_schedule: NetSStoreGasSchedule, computation: ComputationAPI)
 
             if original_value == value:
                 if original_value == 0:
+                    # adds 19800 to refund counter
```

Comment:
nit - feels like it could get stale and be confusing if this number changes in a future fork?
```suggestion
                    # adds SOME_CONSTANT to refund counter
```